pkg/monitors/jenkins/nodemetrics.go

```diff
@@ -0,0 +1,72 @@
+package jenkins
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/signalfx/golib/v3/datapoint"
+	jc "github.com/signalfx/signalfx-agent/pkg/monitors/jenkins/client"
+)
+
+// map key:metric_name value:json_key
+var nodeMetricsMap = map[string]string{
+	"jenkins.node.vm.memory.total.used":    "vm.memory.total.used",
+	"jenkins.node.vm.memory.heap.usage":    "vm.memory.non-heap.used",
+	"jenkins.node.vm.memory.non-heap.used": "vm.memory.non-heap.used",
+	"jenkins.node.queue.size":              "jenkins.queue.size.value",
+	"jenkins.node.health-check.score":      "jenkins.health-check.score",
+	"jenkins.node.executor.count":          "jenkins.executor.count.value",
+	"jenkins.node.executor.in-use":         "jenkins.executor.in-use.value",
+}
+
+type NodeMetrics struct {
+	Version string                            `json:"version"`
```

Comment:
Node version could make a good dimension property.

Reply:
This is not the Jenkins version, I think it might be the metric plugin version.
```
version: "4.0.0"
```
but again, checking the UI, the plugin version is 4.0.2.6

Reply:
```
Detecting Jenkins version
To check the version of Jenkins, load the top page (or, as of 1.483, any .../api/* page too) and check for the X-Jenkins response header. This contains the version number of Jenkins, like "1.404" This is also a good way to check if an URL is a Jenkins URL.
```

version will be added a property over `jenkins_cluster`